src/riak_object.erl

```diff
@@ -419,7 +422,8 @@ fold_contents({r_content, Dict, Value}=C0, MergeAcc, Clock) ->
                     %% value that is the result of a PUT that
                     %% reconciled this sibling value, we can therefore
                     %% (potentialy, see above) drop this value.
-                    MergeAcc#merge_acc{drop=add_drop_candidate(PureDot, C0, Drop)};
+                    MergeAcc#merge_acc{drop=add_drop_candidate(PureDot, C0,
```

Comment:
In the future. def, i just couldn't take all the ill-formatted legacy code. But, agree, *for next time*.